### PR TITLE
add expand collapse style

### PR DESCRIPTION
**Pull request by frrrances**
This PR adds a new expand/collapse pattern for hiding a large block of content (like a long list of item locations). It also adds an unbutton extend to remove default browser button styles when you don't need them.

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -853,6 +978,6 @@
     <li>Vestibulum auctor dapibus neque.</li>
   </ul>
 
-  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque odio soluta fuga esse accusantium, voluptate, eos quod impedit! Quas cum saepe eaque quisquam culpa libero ipsum fuga necessitatibus tempore? Sequi!</p>
+  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque odio soluta fuga esse accusantium, voluptate, eos quod impedit! Quas cum saepe eaque quisquam culpa libero ipsum fuga necessitatibus tempore? Sequi! Sequi!</p>
```
Comment (JPrevost) [inline examples.md]: I assume this is not a functional change?

File: dest/js/libraries-main.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+$(function() {
+  var b = $(".expand-collapse-wrap .button"); // set up all expand buttons
+  var w = $(".expand-collapse-wrap"); // close all expand wraps
+  
+  w.height(100); // customize how much to show
```
Comment (JPrevost) [inline dest/js/libraries-main.js]: This is not blocking, but I'm not sure how to customize or what this number means.

File: examples.md (hunk 1 above)
Comment (frrrances) [reply examples.md]: Sequi! An artifact of testing the Jekyll site compile that I was too lazy to remove.

File: dest/js/libraries-main.js (hunk 2 above)
Comment (frrrances) [reply dest/js/libraries-main.js]: i guess i could put that in the docs

**Pull request merged**